estate_partner/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Estate",
+    "author": "Olga Krikunova",
+    "version": "18.0.1.0.0",
+    "license": "LGPL-3",
+    "depends": ["estate", "base", "partner_autocomplete"],
```

Comment:
Since estate module depends on base, we don't need to inherit the base module if we inherit estate

```suggestion
    "depends": ["estate", "partner_autocomplete"],
```